### DISCUSSION TITLE
How to create "split" screen for multiple lights (SBL#2)

### DISCUSSION THREAD

**Discussion by NoBagels** [maca88/SmartBikeLights#24] [Q&A]
I have two Bontrager Ion Pro RT that I want to control, and thus have installed SBL#2. In the [screenshot](https://apps.garmin.com/en-US/apps/2e27525f-3847-4e49-89af-673970692df6#compatible-devices) shown the screen is split so you can chose light for Ion Pro RT and Flare RT, but I'm not sure how to configure this, and I can't find any info in the wiki?

BR

**Comment by maca88** [comment]
Currently there is no documentation on GitHub for setting up multiple lights, only on Garmin forums (links below). I will add the documentation also here in the following days.

There are two ways on how to control two Bontrager Ion Pro RT lights:

1. By using only one data field, but with the limitation that the configuration for both lights is the same and it is available only for devices with more than 32KB of memory (Check [this post](https://forums.garmin.com/developer/connect-iq/f/showcase/248492/smart-bike-lights-data-field-bike-lights-control-widget/1330764#1330764) for more info)
2. By using two data fields, where you need to set the serial number in order to work (check [this post](https://forums.garmin.com/developer/connect-iq/f/showcase/248492/smart-bike-lights-data-field-bike-lights-control-widget/1328868#1328868) for more info)

If you wanted to have one light panel where you could control both Bontrager Ion Pro RT separately, that is currently not possible. In such case, the closest option would be to have two data fields, both set to an one field layout.

In case you need additional explanation let me know.
